contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
+
+## Which trailers are to be added?
+
+- `Helped-by: Full Name <email@address>`
+   Added by regular users to particular commits within a PR. Applies to the commit it appears on. Means "this commit was somehow improved based on suggestions by @reviewer".
+- `Reviewed-by: Full Name <email@address>`
+   Added by maintainer to PR merge commit. Applies to whole PR. Means "these changes were carefully reviewed by @reviewer an he has stated conformity with them."
+
+## Which tools are included, and how they help?
+
+- Command line tool to transform GitHub usernames into `Full Name <email@address>` info.
+  Script uses GitHub API to look for the info. If user doesn't have a published email, it tries to find a matching one at local git repo. Successful results are cached locally to speed up subsequent requests for the same user.
+  INSTALL: You need curl and jq for this work. Then, put the script in your path.
```

Comment:
perhaps surround `curl` and `jq` with backtics, and provide a link to their home pages:

https://stedolan.github.io/jq/
http://curl.haxx.se/